core/lib/multivm/src/versions/vm_latest/tracers/dispatcher.rs

```diff
@@ -0,0 +1,113 @@
+use crate::interface::dyn_tracers::vm_1_3_3::DynTracer;
+use crate::interface::tracer::{TracerExecutionStatus, VmExecutionStopReason};
+use crate::vm_latest::{
+    BootloaderState, HistoryMode, SimpleMemory, TracerPointer, VmTracer, ZkSyncVmState,
+};
+use zk_evm_1_3_3::tracing::{
+    AfterDecodingData, AfterExecutionData, BeforeExecutionData, VmLocalStateData,
+};
+use zksync_state::{StoragePtr, WriteStorage};
+
+/// Tracer dispatcher is a tracer that can dispatch calls to multiple tracers.
+pub struct TracerDispatcher<S: WriteStorage, H: HistoryMode> {
```

Comment:
Actually, it could be just a new type or even `Vec<Box>>` but for better readability for the first iteration, i've decided to keep it in this way